metadata-db-app/src/cmr/metadata_db/services/generic_documents.clj

```diff
@@ -95,21 +93,24 @@
         orig-native-id (:native-id latest-document)
         orig-concept-id (:concept-id latest-document)
         orig-create-date (:created-at latest-document)
+        doc-name (:document-name latest-document)
         metadata (assoc document-map
-                     :concept-type concept-type
-                     :revision-id (+ latest-rev-id 1)
-                     :native-id orig-native-id
-                     :concept-id orig-concept-id
-                     :revision-date (dtp/clj-time->date-time-str (tkeeper/now))
-                     :created-at orig-create-date)
-        _ (data/save-concept db provider-id metadata)
-        updated (first (data/get-latest-concepts db :generic {:provider-id provider-id} [concept-id]))]
+                        :concept-type :generic
```

Comment:
This can't be generic anymore.  Use the concept type that was defined above
:concept-type concept-type